averages.js

```diff
@@ -0,0 +1,59 @@
+Array.prototype.total = function () {
+  var total = 0;
+  this.forEach(function(num) {
+    total += num;
+  });
+  return total;
+};
+
+Array.prototype.mean = function () {
+  var total = 0;
```

Comment:
You can actually call `this.total()` to use the function you already wrote above!